staging/src/k8s.io/client-go/informers/rbac/v1/clusterrole.go

```diff
@@ -51,23 +52,31 @@ func NewClusterRoleInformer(client kubernetes.Interface, resyncPeriod time.Durat
 	return NewFilteredClusterRoleInformer(client, resyncPeriod, indexers, nil)
 }
 
+func NewClusterRoleInformerWithMultiTenancy(client kubernetes.Interface, resyncPeriod time.Duration, indexers cache.Indexers, tenant string) cache.SharedIndexInformer {
+	return NewFilteredClusterRoleInformerWithMultiTenancy(client, resyncPeriod, indexers, nil, tenant)
+}
+
 // NewFilteredClusterRoleInformer constructs a new informer for ClusterRole type.
 // Always prefer using an informer factory to get a shared informer instead of getting an independent
 // one. This reduces memory footprint and number of connections to the server.
 func NewFilteredClusterRoleInformer(client kubernetes.Interface, resyncPeriod time.Duration, indexers cache.Indexers, tweakListOptions internalinterfaces.TweakListOptionsFunc) cache.SharedIndexInformer {
+	return NewFilteredClusterRoleInformerWithMultiTenancy(client, resyncPeriod, indexers, tweakListOptions, "default")
```

Comment:
metav1.TenantDefault?

and do we still want default tenant?

Reply:
That is the backward-compatible version, where the tenant value is set to "default".

There is new version, ClusterRolesWithMultiTenancy(), which allows the "tenant" to take any value.